### DISCUSSION TITLE
weekly sync-up

### DISCUSSION THREAD

**Discussion by thomas-fossati** [ietf-rats-wg/draft-ietf-rats-corim#38] [General]
# Regular Sync-up

We hold a [weekly meeting on Zoom](https://armltd.zoom.us/j/98186026406?pwd=VnJScC9HUGhmR3ppV2x3VWlQMTdmUT09) every Wednesday, 1500-1600 London.

The routine is we go through the open issues, discuss them and try to converge on actionable outcomes.

The meeting is open to anyone with an interest in CoRIM.

We will publish an agenda here in advance of each meeting.  If you have anything to discuss, feel free to add it to the list.

We don't keep minutes as we typically use the issue tracker to record the result of our discussions.  For more general topics though, this looks like a good place where we can keep track of these "unanchored" threads.

#### Join Zoom Meeting 

https://armltd.zoom.us/j/98186026406?pwd=VnJScC9HUGhmR3ppV2x3VWlQMTdmUT09 

Meeting ID: 981 8602 6406 
Passcode: 806008 

Dial by your location:
+44 203 481 5240 United Kingdom
+1 408 638 0968 US (San Jose)
+1 646 518 9805 US (New York)
+1 346 248 7799 US (Houston)
+91 406 480 2722 India
+91 806 480 2722 India
+91 116 480 2722 India
+91 224 879 8012 India
+852 5803 3730 Hong Kong SAR
+46 8 4468 2488 Sweden
+47 2400 4735 Norway
+972 3 978 6688 Israel
+353 1 536 9320 Ireland
+36 1 408 8456 Hungary
+49 69 3807 9883 Germany
+33 1 7037 2246 France
+358 3 4109 2129 Finland
+45 32 70 12 06 Denmark
+1 438 809 7799 Canada
+82 2 3143 9611 Korea, Republic of
+65 3158 7288 Singapore
+27 87 550 3946 South Africa
+32 1579 5132 Belgium
+48 22 307 3488 Poland
+386 1600 3102 Slovenia
+60 3 3099 2229 Malaysia
+886 (2) 7741 7473 Taiwan
+81 3 4578 1488 Japan

Meeting ID: 981 8602 6406 
Passcode: 806008 
Find your local number: https://armltd.zoom.us/u/asKLJgIHz 

#### Join by SIP 

[98186026406@zoomcrc.com](mailto:98186026406@zoomcrc.com) 

#### Join by H.323 

162.255.37.11 (US West) 
162.255.36.11 (US East) 
115.114.131.7 (India Mumbai) 
115.114.115.7 (India Hyderabad) 
213.19.144.110 (Amsterdam Netherlands) 
213.244.140.110 (Germany) 
103.122.166.55 (Australia Sydney) 
103.122.167.55 (Australia Melbourne) 
209.9.211.110 (Hong Kong SAR) 
149.137.40.110 (Singapore) 
64.211.144.160 (Brazil) 
69.174.57.160 (Canada Toronto) 
65.39.152.160 (Canada Vancouver) 
207.226.132.110 (Japan Tokyo) 
149.137.24.110 (Japan Osaka) 

Meeting ID: 981 8602 6406 
Passcode: 806008

**Comment by henkbirkholz** [comment]
:ship: it
